features/admin.roles.v2/components/readonly-role-list.tsx

```diff
@@ -280,19 +279,6 @@ export const ReadOnlyRoleList: React.FunctionComponent<ReadOnlyRoleListProps> =
         ];
     };
 
-    /**
-     * Resolves data table actions.
-     */
-    const resolveTableActions = (): TableActionsInterface[] => {
```

Comment:
wouldn't removing eye icon break consistency as other UIs use the eye icon to indicate the resource is read only?

Reply:
<img width="1141" alt="Screenshot 2025-02-08 at 00 03 32" src="https://github.com/user-attachments/assets/893daa59-bef1-4685-9581-6dbfb58dd9dc" />

Reply:
This is the same component as of Users > roles.

Reply:
Ack